### PR TITLE
CEDS-2024 disassociate

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: app/views/disassociate_ucr.scala.html [hunk 1]
```diff
@@ -15,45 +15,55 @@
...
 @import forms.DisassociateKind
 
-@this(main_template: views.html.main_template)
+@this(govukLayout: views.html.gds_main_template,
+        govukButton: GovukButton,
+        govukRadios: GovukRadios,
+        exportsInputText: components.gds.exportsInputText,
+        errorSummary: components.errorSummary,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+        appConfig: AppConfig)
```
Comment (PatrykRudnicki) [inline app/views/disassociate_ucr.scala.html]: I know it's a pattern and consistent with rest pages, but appConfig is not used here and can be removed. It'll make testing easier

File: app/assets/stylesheets/customsdecexfrontend-app.scss [hunk 2]
```diff
@@ -19,5 +19,6 @@
 }
 
 $govuk-assets-path: '/customs-movements/assets/';
+$govuk-assets-path: "/customs-movements/assets/lib/govuk-frontend/govuk/assets/";
```
Comment (SteveSugden) [inline app/assets/stylesheets/customsdecexfrontend-app.scss]: Is this a genuine change this time?

File: test/unit/views/DisassociateUcrViewSpec.scala [hunk 3]
```diff
@@ -16,40 +16,56 @@
...
-import views.spec.{UnitViewSpec, ViewMatchers}
+import views.html.disassociate_ucr
 import views.tags.ViewTest
 
 @ViewTest
-class DisassociateUcrViewSpec extends UnitViewSpec with CommonMessages with ViewMatchers {
+class DisassociateUcrViewSpec extends ViewSpec with Injector {
 
-  private val page = new views.html.disassociate_ucr(mainTemplate)
+  private implicit val request = journeyRequest(DisassociateUcrAnswers())
```
Comment (PatrykRudnicki) [inline test/unit/views/DisassociateUcrViewSpec.scala]: Why request is implicit when we're using it explicitly?

Comment (SteveSugden) [reply test/unit/views/DisassociateUcrViewSpec.scala]: I was going to ask something similar - but it is used implicitly elsewhere I think

File: app/views/disassociate_ucr.scala.html (hunk 1 above)
Comment (MaciejRewera) [reply app/views/disassociate_ucr.scala.html]: 👍

File: test/unit/views/DisassociateUcrViewSpec.scala (hunk 3 above)
Comment (MaciejRewera) [reply test/unit/views/DisassociateUcrViewSpec.scala]: messages require a request implicitly.

File: app/assets/stylesheets/customsdecexfrontend-app.scss (hunk 2 above)
Comment (PauloPortugal) [reply app/assets/stylesheets/customsdecexfrontend-app.scss]: Does this fix the font issue?

Comment (MaciejRewera) [reply app/assets/stylesheets/customsdecexfrontend-app.scss]: No. This is wrong. Thank you for spotting this.

**Pull request merged**